submissions/mustbefail/html-movie-seat-booking/index.html

```diff
@@ -12,208 +12,216 @@
+          <img
+            src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg"
+            alt="film poster"
+          />
+        </div>
+        <article class="movie-info">
+          <h1>The Devil All the Time</h1>
+          <p class="tagline">
+            "Everyone ends up in the same <br />
+            damned place."
+          </p>
+          <div class="info-block">
+            <img
+              src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg"
+              alt="film poster"
+            />
+            <div>
+              <div class="rating">
+                <div class="rating-badge">74</div>
+                <span>Viewers rating</span>
+              </div>
+              <span class="rate tag">R</span>
+              <time class="tag">11/09/2020 (US)</time>
+              <time class="tag">2h 18m </time>
+              <span class="tag">Crime, Drama, Thriller</span>
+            </div>
+          </div>
+          <p class="description">
+            In Knockemstiff, Ohio and its neighboring backwoods, sinister
+            characters converge around young Arvin Russell as he fights the evil
+            forces that threaten him and his family. Antonio Campos Director,
+            Screenplay Donald Ray Pollock Novel Paulo Campos Screenplay
+          </p>
+          <div class="trailer">
+            <button>Watch trailer</button>
+          </div>
+        </article>
+        <div class="hall-box">
+          <div class="legend">
+            <div class="description-block">
+              <div class="seat"></div>
+              <span>N/A</span>
+            </div>
+            <div class="description-block">
+              <div class="seat selected"></div>
+              <span>Selected</span>
+            </div>
+            <div class="description-block">
+              <div class="seat occupied"></div>
+              <span>Occupied</span>
+            </div>
+          </div>
+          <div class="screen-block">
+            <div class="screen"></div>
           </div>
-          <article class="movie-info">
-            <h1>The Devil All the Time</h1>
-            <p class="tagline">"Everyone ends up in
-              the same <br> damned place."
-            </p>
-            <div class="info-block">
-              <img src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg" alt="film poster">
-              <div>
-                <div class="rating"><div class="rating-badge">74</div><span>Viewers rating</span></div>
-                <span class="rate">R</span>
-                <span class="date">11/09/2020 (US)</span>
-                <span class="durotaiton">2h 18m </span>
-                <span class="genres">Crime, Drama, Thriller</span>
+          <div class="hall">
+            <form action="#">
+              <div class="row">
+                <label class="seat">
+                  <input type="checkbox" />
```

Comment:
Why you removed `value` and `aria-label` attributes? Also, remove element shifting on `:focus` [watch video](https://share.vidyard.com/watch/56yrhwy832YBSX7PeLbDHi?) Avoid this behavior in the future.